cg/cli/workflow/mutant/base.py

```diff
@@ -100,3 +98,37 @@ def start_available(context: click.Context, dry_run: bool = False):
             exit_code = EXIT_FAIL
     if exit_code:
         raise click.Abort
+
+
+@mutant.command("store-available")
+@DRY_RUN
+@click.pass_context
+def store_available(context: click.Context, dry_run: bool) -> None:
+    """Run QC checks and store bundles for all finished analyses in Housekeeper."""
+
+    analysis_api: MutantAnalysisAPI = context.obj.meta_apis["analysis_api"]
+
+    exit_code: int = EXIT_SUCCESS
+
+    analysis_api.run_qc_and_fail_analyses(dry_run=dry_run)
+
+    for case_obj in analysis_api.get_cases_to_store():
+        LOG.info(f"Storing deliverables for {case_obj.internal_id}")
+        try:
+            store(context=context, case_id=case_obj.internal_id, dry_run=dry_run)
```

Comment:
Is this correct @islean?
Was not sure of how to call store without invoking the cli command. 🤔

Reply:
Looks good to me!

Reply:
In an ideal world the CLI should not really do much, so it is still weird to call the "store" method in another CLI file. But that is an old artifact.